.shellcheckrc

```diff
@@ -0,0 +1,4 @@
+# ShellCheck config file
```

Comment:
Please add this file to the `.structure-config` file.

Reply:
Ah, good catch, sorry!